### PR TITLE
AAP-22221 fixed formatting issues

### PR DESCRIPTION
**Pull request by emurtoug**
Fixed formatting issues in [Planning your automation jobs using the automation savings planner](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/planning_your_automation_jobs_using_the_automation_savings_planner/index#doc-wrapper) using ([AAP-22221](https://issues.redhat.com/browse/AAP-22221))

Affects versions 2.4, 2.3, 2.2

### CODE REVIEW

File: downstream/assemblies/analytics/assembly-automation-savings-planner.adoc [hunk 1]
```diff
@@ -4,11 +4,11 @@ ifdef::context[:parent-context: {context}]
...
+= About the {planner}
 
-The following procedures introduces the ways to get started with your automation savings plans by creating new plans or managing existing plans for your automation needs.
+An automation savings plan gives you the ability to plan, track, and analyze the potential efficiency and cost savings of your automation initiatives. Use {InsightsName} to create an automation savings plan by defining a list of tasks needed to complete an automation job. You can then link your automation savings plans to an Ansible job template in order to accurately measure the time and cost savings upon completion of an automation job.
 
-include::analytics/con-automation-savings-plan.adoc[leveloffset=+1]
+In doing so, you can utilize the {planner} to prioritize the various automation jobs throughout your organization and understand the potential time and cost savings from your automation initiatives.
```
Comment (sayjadha) [inline downstream/assemblies/analytics/assembly-automation-savings-planner.adoc:11]: Should this sentence come right after "...cost savings upon completion of an automation job"? Sounds like a continuation of the concept 'automation savings plan'.

Comment (sayjadha) [inline downstream/assemblies/analytics/assembly-automation-savings-planner.adoc:11]: Suggest 'In doing so' -> 'To create an automation savings plan' for clarity.

Comment (sayjadha) [inline downstream/assemblies/analytics/assembly-automation-savings-planner.adoc:11]: From -> 'of' or 'for'

**Review by sayjadha — COMMENTED** [review]
Minor comments for your attn, @emurtoug. Rest LGTM.

File: downstream/assemblies/analytics/assembly-automation-savings-planner.adoc (hunk 1 above)
Comment (emurtoug) [reply downstream/assemblies/analytics/assembly-automation-savings-planner.adoc]: Thanks @sayjadha! I think utilizing the paragraph break here helps to break up the information and clarify that this sentence is leading into the next module: [Creating a new automation savings plan](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/planning_your_automation_jobs_using_the_automation_savings_planner/index#proc-create-savings-plan_assembly-automation-savings-planner-ctxt). So, I'm inclined to leave it as is. What do you think?

**Pull request merged**